cluster/src/main/java/io/scalecube/cluster/ClusterImpl.java

```diff
@@ -351,4 +352,66 @@ public NetworkEmulator networkEmulator() {
   public boolean isShutdown() {
     return onShutdown.isDisposed();
   }
+
+  public interface ClusterMonitorMBean {
+    String getMember();
+    int getIncarnation();
+    Map<String, String> getMetadata();
+    List<String> getAliveMembers(); // all alive members except local one
```

Comment:
Plz remove `// all alive members except local one` and `// let's keep recent N dead members for full-filling this method`  comments